### DISCUSSION TITLE
docker-compose 安装的sqle，如何升级 SQLE 到最新版本？

### DISCUSSION THREAD

**Discussion by fstln-joe** [actiontech/sqle#725] [Q&A] [answered]
我是用docker-compose安装的SQLE。看了下 SQLE 的文档，没有找到版本升级指南，请问下大家是怎么升级版本的？

**现有版本号**
* UI Version: main 9055e77
* Server Version: main aee23c8c54

**最新版本**
[SQL审核 | SQLE 1.2207.0 来啦！](https://opensource.actionsky.com/20220729-sqle/)

**Comment by HuangWeiCen** [comment, marked as answer]
1. 将sqle rpm包拷贝到容器内
`docker cp {rpm包} {容器ID}:/`
2. 使用root用户登入容器
`docker exec -it -uroot {容器ID}`
3. 和正常升级RPM包安装的SQLE一样升级
https://github.com/actiontech/sqle/discussions/433
4. 退出容器后重新启动容器
`docker restart {容器ID}`

**Comment by sjjian** [comment]
你可以把sqle-server 这个容器删除，然后重新docker pull 下，更新本地的images， 最后使用docker-compose up -d sqle-server。注：在执行上述步骤之前要提前做好数据备份和执行升级脚本
